.github/ISSUE_TEMPLATE/new_IDA_plugin.yml

```diff
@@ -79,6 +79,12 @@ body:
       label: Download SHA256 Hash
       description: |
         SHA256 hash of the file or ZIP downloaded from the download url introduced in the previous field. The hash is used for verification purposes. Example: `62af5cce80dbbf5cdf961ec9515549334a2112056d4168fced75c892c24baa95`
+  - type: input
+    id: files
```

Comment:
@Schamper are you aiming for this field to be used by our automation? Have you tested the GH action [new_package.yml](https://github.com/mandiant/VM-Packages/blob/main/.github/workflows/new_package.yml#L22) with this change? I think it does not work, as we are using `template-path: .github/ISSUE_TEMPLATE/new_package.yml` to parse the issue and I think fields not included there are ignored.